### PR TITLE
fix: Stale bot configuration changes

### PR DESCRIPTION
**Pull request by kuja53**
# Describe what this PR does #

As I mention on slack channel. I check configuration of stale probot and add global stale and close period because it seems that actual version of probot needs to have this addressed in stale.yml file. And also rename it from stale.yaml to stale.yml.

### CODE REVIEW

**Comment by humblec** [comment]
Thanks for the PR @kuja53 ! :+1:

**Comment by humblec** [comment]
@kuja53 just need some adjustment to the commit message to make travis happy

```
 commitlint-travis

⧗   input: Rename file to match actual needs of probot.

Signed-off-by: Jakub Kuzelka <jakub@kuzelka.pw>

✖   header may not end with full stop [header-full-stop]

✖   found 1 problems, 0 warnings

ⓘ   Get help: https://github.com/conventional-changelog/commitlint/#what-is-commitlint

⧗   input: Add global stale and close period to be sure that bot address separate resources right.

Signed-off-by: Jakub Kuzelka <jakub@kuzelka.pw>

✖   header must not be longer than 72 characters, current length is 87 [header-max-length]

✖   header may not end with full stop [header-full-stop]

✖   found 2 problems, 0 warnings

ⓘ   Get help: https://github.com/conventional-changelog/commitlint/#what-is-commitlint
```

File: .github/stale.yml [hunk 1]
```diff
@@ -2,13 +2,20 @@
...
----
+
 exemptLabels:
   - keepalive
   - security
   - reliability
   - release requirement
 
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
```
Comment (Madhu-1) [inline .github/stale.yml:13]: This value is configured on both issue https://github.com/ceph/ceph-csi/blob/a2c108159172c68e9b66c9844b5c526451db9439/.github/stale.yml#L33-L34 and PR https://github.com/ceph/ceph-csi/blob/a2c108159172c68e9b66c9844b5c526451db9439/.github/stale.yml#L44-L45  and this configuration is working fine in rook https://github.com/rook/rook/blob/master/.github/stale.yml @travisn or @leseb can you please confirm

**Review by Madhu-1 — COMMENTED** [review]
For me looks like only renaming should be enough but need to be tested it

**Comment by kuja53** [comment]
Renaming it its enough. But I check it in issues that sometimes bot can not detect stale and close baseline if you have not global values. Its also my experience on another project. But its not necessarry in general. We will see.

**Comment by humblec** [comment]
> Renaming is enough. But I check it in issues that sometimes bot can not detect stale and close baseline if you have not global values. Its also my experience on another project. But its not necessarry in general. We will see.

@kuja53 sure, can you please correct the commit message to make CI happy. We can get this merged as soon its done.

**Comment by kuja53** [comment]
Sorry, I'm overlooked the errors. I fix it now.

**Review by humblec — APPROVED** [review]

File: .github/stale.yml [hunk 2]
```diff
@@ -9,6 +9,12 @@ exemptLabels:
   - reliability
   - release requirement
 
+# Number of days of inactivity before stale
+daysUntilStale: 60
+
+# Number of days of inactivity before close
+daysUntilClose: 30
```
Comment (humblec) [inline .github/stale.yml:16]: @kuja53 can we reduce this to 7 please?

**Comment by humblec** [comment]
Lets get this merged to see stale bot comes in action or not.. @Madhu-1 any final comments?

File: .github/stale.yml (hunk 2 above)
Comment (Madhu-1) [reply .github/stale.yml:16]: This need to be reduced total 90 days seems high

**Review by nixpanic — CHANGES_REQUESTED** [review]
As mentioned by others, please change the `daysUntil*` values a little.

**Comment by kuja53** [comment]
@nixpanic Ok, and what is the right value from your point of view? I made it as implicite value from my experience on another projects.

**Review by kuja53 — COMMENTED** [review]
Ok, no problem. I will update it with new values.

**Review by kuja53 — COMMENTED** [review]
I remove global values to not mess between issues and pulls differences.

**Review by humblec — APPROVED** [review]

**Review by nixpanic — COMMENTED** [review]
This could have been done in one or two commits. Do you want to cleanup the history by squashing and force pushing the commits?

**Comment by kuja53** [comment]
> This could have been done in one or two commits. Do you want to cleanup the history by squashing and force pushing the commits?

I squash all commits in to one and rebease it :) You are right.

**Comment by nixpanic** [comment]
> > This could have been done in one or two commits. Do you want to cleanup the history by squashing and force pushing the commits?
> 
> I squash all commits in to one and rebase it :) You are right.

Great, thanks! However you should now adjust the commit message a little, much repetition in there :disappointed:

**Review by humblec — APPROVED** [review]

**Review by nixpanic — APPROVED** [review]

**Pull request merged**